mobile/src/screens/meditation-home/libs/constants.ts

```diff
@@ -0,0 +1,6 @@
+const navigationItems = [
```

Comment:
Follow quality criteria

Reply:
@v0ldemar01 could you please explain whats wrong with it ? we have the same name on frontend and B18. Arrays are named plural nouns.

Reply:
@iamAlinaaa B22. Constant names are written in capital letters.

Reply:
@danilittaa mockedData was written like that thats why im asking ) and i saw examples for  B22 : 
const MAX_HEIGHT = 6996; 
const IDX_NOT_FOUND = -1;   
so i just thought array is not related to this 🤔 . because for arrays we have B18. Arrays are named plural nouns. (and in examples: 
const ages = [10, 15, 22];
const names = ['John', 'Pit', 'Brew'];

const cat = {
  name: 'Pit',
  friends: ['Nike', 'Sof', 'Kat'],
};